frontend/summary/dataPivot/DPFOrderTab.js

```diff
@@ -371,7 +335,7 @@ let buildFilterTable = function(tab, dp, handleTableChange) {
 
         buildFilterTable(tab, dp, handleTableChange.bind(this));
         buildFilterBooleanDiv(tab, dp, handleTableChange.bind(this));
-        buildSortingTable(tab, dp, handleTableChange.bind(this));
```

Comment:
Since `handleTableChange` was removed the row customization part of the form is no longer forced to refresh when order changes.

Reply:
thanks; this is now handled just as it was before